modules/interop.nix

```diff
@@ -18,18 +14,6 @@ with builtins; with lib;
       default = true;
       description = "Include Windows PATH in WSL PATH";
     };
-
-    preserveArgvZero = mkOption {
```

Comment:
Should we add a mkRemovedOption?

Reply:
It's an internal option so I don't know.